### DISCUSSION TITLE
貌似软件不支持nftable？

### DISCUSSION THREAD

**Discussion by zzz6839** [UnblockNeteaseMusic/luci-app-unblockneteasemusic#168] [Q&A] [answered]
在使用immortalwrt 最新的commit 35bf918edf8422af8940767aaea3a9a95dde9e76时发现Unblockneteasemusic无法接管局域网内的设备，不知道未来是否有跟进计划

**Comment by 1715173329** [comment, marked as answer]
已于 20 日前修复，请下载最新版本。
